### ISSUE TITLE
Question about the perm field and antennas

### ISSUE THREAD

**Issue by daniellejeong** [dhalperi/linux-80211n-csitool-supplementary#322]
I'm not sure if I understood the perm field correctly, so I just want to clarify it!

In FAQ page, it says
**perm** tells us how the NIC permuted the signals from the 3 receive antennas into the 3 RF chains that process the measurements. The sample value of [3 2 1] implies that Antenna C was sent to RF Chain A, Antenna B to Chain B, and Antenna A to Chain C. This operation is performed by an antenna selection module in the NIC and generally corresponds to ordering the antennas in decreasing order of RSSI.

So if there are 1 transmitter and 3 receiver antennas, 
I would get a time series data of 1x3x30, which is 3 streams of data of the 30 subcarriers.

Does the statement about perm mean that 
the 3 streams of CSI data is not actually the CSI stream of corresponding antenna A,B,C?
In other words, the first stream may contain the CSI data of one of the three antennas A,B,C depending on the RSSI value?

I'm confused about the issue because the 3 streams data look so different from each other.
The antennas are closely located, but they show such different pattern and magnitude range,
so I was trying to figure out why this is so.
The data in each stream seems to have consistent magnitude range, so I thought each stream corresponds to each antenna.

If I want to see the CSI data of each antenna, 
do I have to rearrange and permute the CSI packets with the values in the perm field?

Thank you so much in advance!

**Comment by titinho11** [comment]
Hello @daniellejeong, I ask myself the same question. Have you had an answer?
